src/redux/reducers/cellInfo/index.js

```diff
@@ -9,15 +11,15 @@ const cellInfoReducer = (state = initialState, action) => {
     case CELL_INFO_UPDATE: {
       return cellInfoUpdate(state, action);
     }
-
     case CELL_INFO_FOCUS: {
       return cellInfoFocus(state, action);
     }
-
     case CELL_INFO_UNFOCUS: {
       return cellInfoUnfocus(state, action);
     }
-
+    case EXPERIMENT_SETTINGS_PIPELINE_START: {
+      return initialState;
+    }
```

Comment:
I understand your approach here, but I have a concern that in the future if we write another part of the redux store, the developer who writes it might forget about adding these lines to invalidate the store. Do you agree that this can be a concern and if so do you have any ideas of what we can do about it?

Reply:
Yes, good point. The problem is that you cannot just reset /everything/ to its initial state. If that was the case, I would have just done it in the root reducer and be done with it, but the Redux store also includes things like the layout of the pages, experiment details, authentication credentials, etc. You cannot automatically check for the presence of this because whether you need to reset a certain store (or sometimes, only part of a store) is dependent on context.